.github/workflows/build-python.yml

```diff
@@ -12,6 +12,8 @@ jobs:
       CTEST_PARALLEL_LEVEL: 2
       CMAKE_BUILD_TYPE: ${{ matrix.build_type }}
       PYTHON_VERSION: ${{ matrix.python_version }}
+      PYTHON_MAJOR_VERSION: 3
```

Comment:
Why two variables? Also here note that PYTHON_VERSION is a variable in the build matrix.

Reply:
Setting a more specific python version for CMake seemed like a better option, and helps us debug the lowest supported version in the future (helped me figure out the MANIFEST.in issue).

Problem is, that env variable is also used to set the Python interpreter, so it leads to `python3.6.9` which is invalid, hence I added the `PYTHON_MAJOR_VERSION` flag.